src/gaurav/BankSystem.java

```diff
@@ -0,0 +1,77 @@
+package gaurav;
+
+public class BankSystem {
+		String name;
+		int totalBalance;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.